Package.swift

```diff
@@ -199,8 +199,6 @@ let package = Package(
       url: "https://github.com/SlaunchaMan/GCDWebServer.git",
       .revision("935e2736044e71e5341663c3cc9a335ba6867a2b")
     ),
-    // Branches need a force update with a run with the revision set like below.
```

Comment:
Nit: is this comment still helpful to put somewhere?

Reply:
I'm not sure it's still true and I've been working with Gran on better approaches to incremental testing.